app/loc/lang-sv.json

```diff
@@ -165,26 +165,27 @@
-    "STR_INFO_MTG_INACTIVE": "Ärendets status kan ändräs ändast om sammanträdet är under behandling",
+    "STR_INFO_SALI_MODE": "Funktionen är inte möjlig då salsystemet är i bruk",
+    "STR_INFO_MTG_INACTIVE": "Ärendets status kan ändras endast om sammanträdet pågår",
     "STR_INFO_MTG_BAD_STATUS": "Sammanträdets status kan ändras ändast om status är: 'Tekniskt öppnat', 'Pågår' eller 'Avbrutet'",
-    "STR_INFO_TOPIC_STATUS_CHANGE_DISABLED": "Ärendets status kan ändras ändast om inga andra ärende pågår",
-    "STR_HELP_MTGLIST": "Listassa näet kaikki kokouksesi aikajärjestyksessä. Menneet kokoukset löytyvät omalta välilehdeltä.<BR>Voit avata kokouksen painamalla rivillä  painiketta, jossa lukee kokouksessa sallittu käyttäjäroolisi.<BR>Painikkeita voi samalla rivillä olla useita, mikäli kokous sallii sinulle monta roolia. Valitse silloin haluamasi rooli painamalla vastaavaa painiketta.",
-    "STR_HELP_SIGNLIST": "Listassa näet kaikki allekirjoitettavat asiakirjasi. Toimielimien pöytäkirjat ja viranhaltijapäätökset ovat peräkkäin omina ryhminään.<BR>Käsitellyt asiakirjat löytyvät omalta välilehdeltä.<BR>Voit avata asiakirjan painamalla rivillä ainiketta.",
-    "STR_HELP_SIDEMENU": "Voit vaihdella sivuvaliko eri leveyksien välillä käyttäen tätä painiketta",
-    "STR_HELP_MENU_HDR": "Näkymän asettelun toiminnot",
-    "STR_HELP_VIEWMODE": "Näkymän sisältöä esitetään sivuvalikossa sekä kahdessa eri palstassa. Näin voit tarkastella esimerkiksi kokouksen esitystä ja liitettä vierekkäin. Voit vaihtaa palstojen esitystavan joko vierekkäin tai allekkain.",
-    "STR_HELP_MTG_STATES_HDR": "Tärkeimmät kokouksen tilat",
+    "STR_INFO_TOPIC_STATUS_CHANGE_DISABLED": "Ärendets status kan ändras ändast om inga andra ärende är under behandling",
+    "STR_HELP_MTGLIST": "I listan ser du alla dina sammanträden i tidsordning. De tidigare sammanträdena ser du i en egen flik. Du kan öppna sammanträdet genom att klicka på knappen med din tillåtna användarroll vid sammanträdet. Det kan finns många knappar på samma rad, om du kan inneha flera roller på sammanträdet. Välj då den roll du önskar genom att klicka på motsvarande knapp.",
+    "STR_HELP_SIGNLIST": "I listan ser du alla dina dokument som kan undertecknas. Organets protokoll och tjänsteinnehavarbeslut är efter varandra som egna grupper. De behandlade dokumenten finns under en egen flik. Du kan öppna ett dokumentet genom att klicka på en knapp i raden.",
+    "STR_HELP_SIDEMENU": "Du kan ändra på sidmenyns bredd med hjälp av den här knappen.",
+    "STR_HELP_MENU_HDR": "Funktioner för vyns layout",
+    "STR_HELP_VIEWMODE": "Innehållet i vyn visas upp i sidmenyn och i två olika spalter. Du kan välja visningstyp för spalterna antingen bredvid eller ovanpå varandra.",
+    "STR_HELP_MTG_STATES_HDR": "Viktigaste statusar för sammanträdet",
     "STR_HELP_MTG_STATES_BODY": "Ehdotuksia voi julkaista kaikissa kokouksen tiloissa. Aloitteita voi jättää sähköisesti vain Kaupunginvaltuuston kokouksissa.",
```

Comment:
This is still unstranslated, could you ask what they want for this help dialog string?